### PR TITLE
Document missing Callable methods

### PR DESCRIPTION
**Pull request by HaSa1002**
Documents:
 * Callable.hash()
 * Callable.is_custom()
 * Callable.is_null()
 * Callable.is_standard()

### CODE REVIEW

File: doc/classes/Callable.xml [hunk 1]
```diff
@@ -69,24 +69,28 @@
...
 			</return>
 			<description>
+				Returns the hash value of this [Callable]'s object.
 			</description>
 		</method>
 		<method name="is_custom">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [Callable] is a custom callable thats behaviour is different based on implementation details.
```
Comment (Calinou) [inline doc/classes/Callable.xml:79]:
```suggestion
				Returns true if [Callable] is a custom callable whose behaviour differs based on implementation details.
```

Also, I don't find that description isn't very clear. Can it be made more detailed?

**Review by Calinou — DISMISSED** [review]

File: doc/classes/Callable.xml (hunk 1 above)
Comment (HaSa1002) [reply doc/classes/Callable.xml:79]: I know, but there is literally no information about CustomCallables (none evident from the source). My guess is that you use it when you don't have an object pointer or object id and/or need some sort of custom behaviour. Also, get_custom is not exposed to script, so I guess the method is almost useless in a script level. Any ideas?

Comment (HaSa1002) [reply doc/classes/Callable.xml]: I think the whole purpose of the method (without the ability to get the CustomCallable) is to state whether or not `get_method` can be used.

**Comment by HaSa1002** [comment]
I updated the descriptions and added the remaining as well.

File: doc/classes/Callable.xml [hunk 2]
```diff
@@ -67,6 +67,7 @@
 			<return type="Callable">
 			</return>
 			<description>
+			Returns a copy of this [Callable] with the arguments bound. Bound arguments are passed after the supplied by [method call].
```
Comment (akien-mga) [inline doc/classes/Callable.xml:70]: Should have one more level of indentation (running `godot --doctool .` should fix it).

Comment (akien-mga) [reply doc/classes/Callable.xml:70]: It's not so clear so "are passed after the supplied by `call`" - the supplied what?

File: doc/classes/Callable.xml [hunk 3]
```diff
@@ -108,24 +109,28 @@
...
 			</return>
 			<description>
+				Returns the hash value of this [Callable]'s object.
 			</description>
 		</method>
 		<method name="is_custom">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [Callable] is a custom callable whoose behaviour differs based on implementation details. Custom callables are used in the engine for various reasons. If true, you can't use [method get_method].
```
Comment (akien-mga) [inline doc/classes/Callable.xml:119]:
```suggestion
				Returns [code]true[/code] if this [Callable] is a custom callable whose behavior differs based on implementation details. Custom callables are used in the engine for various reasons. If [code]true[/code], you can't use [method get_method].
```

File: doc/classes/Callable.xml [hunk 4]
```diff
@@ -108,24 +109,28 @@
...
 			</return>
 			<description>
+				Returns true if this [Callable] has not target to call the method on.
 			</description>
 		</method>
 		<method name="is_standard">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [Callable] is a standard callable, referencing an object and a method using a [StringName].
```
Comment (akien-mga) [inline doc/classes/Callable.xml:133]:
```suggestion
				Returns [code]true[/code] if the [Callable] is a standard callable, referencing an object and a method using a [StringName].
```

File: doc/classes/Callable.xml [hunk 5]
```diff
@@ -134,6 +139,7 @@
 			<argument index="0" name="right" type="Callable">
 			</argument>
 			<description>
+			Returns true if both [Callable]'s invoke different targets.
```
Comment (akien-mga) [inline doc/classes/Callable.xml:142]:
```suggestion
				Returns [code]true[/code] if both [Callable]s invoke different targets.
```

File: doc/classes/Callable.xml [hunk 6]
```diff
@@ -142,6 +148,7 @@
 			<argument index="0" name="right" type="Callable">
 			</argument>
 			<description>
+			Returns true if both [Callable]'s invoke the same custom target.
```
Comment (akien-mga) [inline doc/classes/Callable.xml:151]:
```suggestion
				Returns [code]true[/code] if both [Callable]s invoke the same custom target.
```

File: doc/classes/Callable.xml [hunk 7]
```diff
@@ -150,6 +157,7 @@
 			<argument index="0" name="argcount" type="int">
 			</argument>
 			<description>
+			Returns a copy of this [Callable] with the arguments unbound. Calling the returned [Callable] will call the method without the extra arguments that are supplied in the [Callable] on which you are calling this method.
```
Comment (akien-mga) [inline doc/classes/Callable.xml:160]:
```suggestion
				Returns a copy of this [Callable] with the arguments unbound. Calling the returned [Callable] will call the method without the extra arguments that are supplied in the [Callable] on which you are calling this method.
```

File: doc/classes/Callable.xml (hunk 2 above)
Comment (HaSa1002) [reply doc/classes/Callable.xml:70]: arguements... Was perhaps a little late

**Comment by HaSa1002** [comment]
applied suggestions and rebased.

File: doc/classes/Callable.xml (hunk 2 above)
Comment (akien-mga) [reply doc/classes/Callable.xml:70]: Still needs an extra level of indentation.

**Review by akien-mga — APPROVED** [review]
Last formatting nitpick, otherwise seems good.

**Comment by HaSa1002** [comment]
I forgot to stage my changes 🤦. It's done now.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**